<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderWindow</class>
 <widget class="QWidget" name="RenderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Render Project</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QSpinBox" name="xResSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>  Y resolution:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QSpinBox" name="yResSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QComboBox" name="unitSizeCB"/>
     </item>
     <item row="0" column="12" rowspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="zoomOutTB">
         <property name="toolTip">
          <string>Zoom out</string>
         </property>
         <property name="text">
          <string>Zoom out</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/NemesisCore/window/RenderWindow/viewmag-.png</normaloff>:/images/NemesisCore/window/RenderWindow/viewmag-.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoomInTB">
         <property name="toolTip">
          <string>Zoom in</string>
         </property>
         <property name="text">
          <string>Zoom in</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/NemesisCore/window/RenderWindow/viewmag+.png</normaloff>:/images/NemesisCore/window/RenderWindow/viewmag+.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoom1t1TB">
         <property name="toolTip">
          <string>Original size</string>
         </property>
         <property name="text">
          <string>1 : 1</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/NemesisCore/window/RenderWindow/viewmag1.png</normaloff>:/images/NemesisCore/window/RenderWindow/viewmag1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoomFitTB">
         <property name="toolTip">
          <string>Zoom to fit</string>
         </property>
         <property name="text">
          <string>Fit to window</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/NemesisCore/window/RenderWindow/viewmagfit.png</normaloff>:/images/NemesisCore/window/RenderWindow/viewmagfit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="copyTB">
         <property name="toolTip">
          <string>Copy to clipboard</string>
         </property>
         <property name="text">
          <string>Copy to clipboard</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/NemesisCore/window/RenderWindow/editcopy.png</normaloff>:/images/NemesisCore/window/RenderWindow/editcopy.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="10" rowspan="2">
      <widget class="QComboBox" name="unitResolutionCB"/>
     </item>
     <item row="0" column="6" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>  X resolution:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="QToolButton" name="sizeLockTB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Keep ratio</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/NemesisCore/window/RenderWindow/encrypted.png</normaloff>:/images/NemesisCore/window/RenderWindow/encrypted.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="9" rowspan="2">
      <widget class="QToolButton" name="resLockTB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Keep ratio</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/NemesisCore/window/RenderWindow/encrypted.png</normaloff>:/images/NemesisCore/window/RenderWindow/encrypted.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSpinBox" name="widthSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSpinBox" name="heightSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollArea {
	background-color: #CCCCCC;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>14</y>
        <width>480</width>
        <height>272</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="notifyLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
